README.md

```diff
@@ -46,6 +46,9 @@ This is a useful tool for Librarians that helps to record what books are in the
  ### Key Features <a name="key-features"></a> 
   
  The tool allows librarians to 
+- Show persons.
```

Comment:
- [ ] Kindly add tests for the Person class. Remember to add tests for custom `add_rentals` and `can_use_services?` methods.